src/boot/bootctl.c

```diff
@@ -1647,6 +1653,18 @@ static int parse_argv(int argc, char *argv[]) {
                         arg_arch_all = true;
                         break;
 
+                case ARG_EFI_LABEL:
+                        if (isempty(optarg) || !string_is_safe(optarg)) {
+                                _cleanup_free_ char *escaped = NULL;
+                                escaped = cescape(optarg);
+                                return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
+                                                       "Invalid --efi-label=: %s", cescape(escaped));
```

Comment:
Please do not double escape the string. Please use `strna()`: `cescape(escaped)` -> `strna(escaped)`.